src/applications/appeals/onramp/tests/cypress/results-shortcuts/results-1-1B.cypress.spec.js

```diff
@@ -31,7 +29,7 @@ describe('Decision Reviews Onramp', () => {
 
       // RESULTS
       h.verifyUrl(ROUTES.RESULTS);
-      h.verifyText(h.RESULTS_HEADER, RESULTS_1_1B);
+      h.verifyNonDrResultsHeader();
```

Comment:
Could we maybe add a `data-test-id` attribute corresponding to the result short name to the root element of each bodyContent object in RESULTS_CONTENT, and verify it here? Feels like otherwise we are losing some specificity/coverage since the specs only check that the header is nonDR or DR.